man/generateRWrappers.Rd

```diff
@@ -0,0 +1,120 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/PythonPkgWrapperUtils.R
+\name{generateRWrappers}
+\alias{generateRWrappers}
+\title{Generate R wrappers for Python classes and functions}
+\usage{
+generateRWrappers(pyPkg, module, setGenericCallback, modifyFunctions = NULL,
+  modifyClasses = NULL, functionPrefix = NULL, pySingletonName = NULL,
+  transformReturnObject = NULL)
+}
+\arguments{
+\item{pyPkg}{The Python package name}
+
+\item{module}{The name of the Python module to be wrapped.}
+
+\item{setGenericCallback}{The callback to setGeneric defined in the target R package}
+
+\item{modifyFunctions}{Optional function to modify the returned functions}
```

Comment:
Optional function to intercept and modify the auto-generated function metadata, including name, docstring, ????.  See example below.